transform.js

```diff
@@ -38,9 +38,9 @@ var CARELINK_TREND_TO_NIGHTSCOUT_TREND = {
   }
 };
 
-function parsePumpTime(pumpTimeString, offset, medicalDeviceFamily) {
+function parsePumpTime(pumpTimeString, offset, offsetMilliseconds, medicalDeviceFamily) {
   if (process.env['MMCONNECT_SERVER'] === 'EU' || medicalDeviceFamily === 'GUARDIAN') {
```

Comment:
The offset issues were affecting me (with MCONNECT_SERVER: US) so I added "or US" to this to get the same fix.  I'm troubleshooting intermittent other issues but I'm guessing those are related to deploying on kubernetes, since it seems fine running in dev.

https://github.com/mattster98/minimed-connect-to-nightscout/commit/21e07caa5e8c4efcf5fa27b3079d9d7f45efaf9b

A more elegant solution (assuming appending the tz offset is still needed in some situations) might be to see if the first method works, and if so, go with it.  If it doesn't, use the second method.  I'm guessing medtronic changed how they're sending the device time out, but not sure.

Reply:
I think US and EU are doing the same thing, and the separation is a historical artifact we can ignore.

I've been meaning to more carefully to emulate what's happening in XDripCarelinkFollower: https://github.com/benceszasz/xDripCareLinkFollower/tree/master/app/src/main/java/com/eveningoutpost/dexdrip/cgm/carelinkfollow.

Reply:
Also note, Sulka disabled some additional datetime munging just recently here: https://github.com/nightscout/cgm-remote-monitor/pull/7809/commits/c1de8a5d84340beee9c2178dc6b9a2fb9b93ada7